app/models/concerns/taxonomix.rb

```diff
@@ -58,36 +48,26 @@ def used_taxonomy_ids
     end
 
     def used_location_ids
-      enforce_default
-      return [] unless which_location && SETTINGS[:locations_enabled]
+      return [] unless which_location
       get_taxonomy_ids(which_location, which_ancestry_method)
     end
 
     def used_organization_ids
-      enforce_default
-      return [] unless which_organization && SETTINGS[:organizations_enabled]
+      return [] unless which_organization
       get_taxonomy_ids(which_organization, which_ancestry_method)
     end
 
     def get_taxonomy_ids(taxonomy, method)
       Array(taxonomy).map { |t| t.send(method) + t.ancestor_ids }.flatten.uniq
     end
 
-    # default scope is not called if we just use #scoped therefore we have to enforce quering
-    # to get correct default values
-    def enforce_default
-      self.where(nil).limit(0).all unless which_ancestry_method.present?
-    end
-
     def taxable_ids(loc = which_location, org = which_organization, inner_method = which_ancestry_method)
       # Return everything (represented by nil), including objects without
       # taxonomies. This value should only be returned for admin users.
       return nil if any_context?(loc) && any_context?(org) &&
         User.current.try(:admin?)
 
-      ids = unscoped.pluck(:id)
```

Comment:
Since we always have taxonomies enabled and not ignored, we save one query here which was executed just to handle the case where both were disabled.